sapi/fpm/fpm/fpm_sockets.h

```diff
@@ -12,9 +12,9 @@
 #include "fpm_worker_pool.h"
 
 /*
-  On FreeBSD and OpenBSD, backlog negative values are truncated to SOMAXCONN
+  On FreeBSD, OpenBSD and macOS, backlog negative values are truncated to SOMAXCONN
```

Comment:
Looks like we forgot to update comment for Linux addition: https://github.com/php/php-src/pull/8410/files. Could you add it when you are in it.